src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -510,7 +510,8 @@ You can pass only one of the following values at a time: `exists`, `values` or `
 `<survey>` contains the following fields:
 
 - `type` (`string`) - allowed values: `pre_chat`, `post_chat`
-- `answer_id` (`string`)
```

Comment:
Hello @MaciejSzaflik :) We have been reported on [Slack](https://livechat.slack.com/archives/C2XMTMQ5S/p1656325767216989) that description of a filter `filters.surveys.<survey>` for `List Archives` in API 3.4 & 3.5, should be updated according to the change in [API-9679](https://livechatinc.atlassian.net/browse/API-9679). Could you please confirm that the introduced changes are done correctly? (I also marked you in the comment, including the change for v3.5. Thanks in advance :)

Reply:
Survey filter should have the same description as here https://github.com/livechat/livechat-public-docs/blob/master/src/pages/data-reporting/reports-api/index.mdx
so I think something like this will work (I changed the description a little bit to be more similar to other filter descriptions in list archive method, than in reports):
```
`<survey>` contains the following fields:

- `from` (`string`) - date & time format with a resolution of microseconds, UTC string, YYYY-MM-DDTHH:MM:SS.ssssssZHH:MM
- `to` (`string`) - date & time format with a resolution of microseconds, UTC string, YYYY-MM-DDTHH:MM:SS.ssssssZHH:MM
- `exists` (`bool`) - if set to `true`, will return only surveys with `survey_id`
- `type` (`string`) - allowed values: `pre_chat`, `post_chat`
- `values` (`string[]`) - an array of answer_ids
- `exclude_values` (`string[]`) - an array of answer_ids
- `groups.values` (`int[]`) - an array of group IDs
- `groups.exclude_values` (`int[]`) - an array of group IDs
```

Reply:
@MaciejSzaflik should be fine now :) If yes, I would appreciate if you could approve the PR :)